src/main/java/fr/pickaria/pterodactylpoweraction/ConnectionListener.java

```diff
@@ -184,8 +194,8 @@ private boolean isReachable(RegisteredServer server) {
             // FIXME: This may be blocking the main thread
             return configurationLoader.getOnlineChecker(server).isRunningNow();
         } catch (NoSuchElementException exception) {
-            logger.error("Server '{}' does not have its Pterodactyl ID configured in the plugin's configuration", server.getServerInfo().getName(), exception);
-            return false;
+            logger.debug("Server '{}' does not have its Pterodactyl ID configured in the plugin's configuration", server.getServerInfo().getName());
+            return new PingOnlineChecker(server, configurationLoader.getConfiguration()).isRunningNow();
```

Comment:
Not sure about this one, if the server is not configured, we may want to bypass all the plugin's logic. Then we should simply return `true` here.